ci/build_cpp.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+gpuci_logger "Get env"
+env
+
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
+gpuci_logger "Check versions"
+python --version
+$CC --version
+$CXX --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+################################################################################
+# BUILD - Conda package builds (LIBRMM)
+################################################################################
+
+CONDA_BLD_DIR=".conda-bld"
+FILE_NAME="conda_librmm_build_${BRANCH_NAME}-arc-${ARC}.tar"
+
+# Build
+conda build \
+  --croot ${CONDA_BLD_DIR} \
+  --no-build-id \
```

Comment:
The `--no-build-id` flag is going to be used for every repository (it's required for `sccache`), so ultimately we should move this to the `.condarc` file for our CI images as mentioned here

- https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#disable-per-build-folder-creation-conda-build-3-0-set-build-id